config/resource-manager.hcl

```
@@ -67,6 +67,10 @@ service "batch" {
   name      = "Batch"
   available = ["2023-05-01", "2023-11-01"]
 }
+service "billing" {
+  name      = "billing"
```

This wants to be Title Cased, which is why the unit tests are failing - mind updating this?

```suggestion
  name      = "Billing"
```

---

Oops, thanks for catching the casing issue @tombuildsstuff. I've added the suggested update now so hopefully those tests should pass and this will be good to go!